src/js/components/Box/StyledBox.js

```diff
@@ -194,6 +194,11 @@ const interactiveStyle = css`
     ${(props) =>
       props.hoverIndicator &&
       getHoverIndicatorStyle(props.hoverIndicator, props.theme)}
+    ${(props) =>
+      props.context === 'card' &&
+      props.onClick &&
+      props.theme.card.hover?.container?.elevation &&
```

Comment:
This feels like Box knows too much about Card. Would it be possible to have Card pass `hoverIndicator` as a property to Box with it pulling the value from `theme.card.hover?`?

Reply:
This feels like a fair approach. Let me try that out.